AndelaOSP/Andela-Socials
# [Feature #159313845] Implement graphql calls

#### What Does This PR Do?
- Adds calls to the graphql end points 
- Stores the response data in the redux store

#### Description Of Task To Be Completed
Adds calls to the graphql end points and stores the response data in the redux store

#### Any Background Context You Want To Provide?
Previously the application was using axios to access data from the Rest API. Since the API is being moved to use graphql, the application will then use Apollo Client to retrieve the data from the API as a redux action and then call a reducer to store the data in the redux store.

#### How can this be manually tested?
One should run the application and then open the app in the browser using `{front-end-url-path}/TestActions` and can check for the changes in the redux store using the redux-dev-tools.

#### What are the relevant pivotal tracker stories?
(#159313845)[https://www.pivotaltracker.com/story/show/159313845]
